website_sale_product_attribute_value_filter_existing/tests/test_website_sale_product_attribute_value_filter_existing.py

```diff
@@ -71,13 +72,10 @@ def setUp(self):
 
     def test_ui_website(self):
         """Test frontend tour."""
-        tour = (
-            "odoo.__DEBUG__.services['web_tour.tour']",
+
+        self.start_tour(
+            "/",
             "website_sale_product_attribute_value_filter_existing",
-        )
-        self.browser_js(
-            url_path="/",
-            code="%s.run('%s')" % tour,
-            ready="%s.tours['%s'].ready" % tour,
+            step_delat=100,
```

Comment:
Is it needed anyway?
```suggestion
            step_delay=100,
```

Reply:
yes, I will keep this to make sure there is enough time between steps. Some tests in Odoo modules are still using this